## Big Data: Checkpoint 3

In this assignment, you'll continue working with the Credit Card Fraud Detection dataset from Kaggle. So, you should load the dataset before working on the following tasks. 
Please submit your solutions to the following tasks as a link to your jupyter notebook on Github.

In [1]:
import dask.dataframe as dd

df = dd.read_csv('data/creditcard.csv', dtype={'Time': 'float64'})

### 1. How many transactions do we have in total?

In [2]:
len(df)

284807

There are 284,807 transactions in total. 

### 2. How many transactions are fraud and how many are not?

In [3]:
df.groupby('Class')['Time'].count().compute()

Class
0    284315
1       492
Name: Time, dtype: int64

Of all the transactions, 492 are fraud, and the rest of the transactions are non-fraudulent.

### 3.  What is the max amount in fraud transactions?

In [4]:
df[df.Class ==1]['Amount'].max().compute()

2125.87

The maximum fraud transaction amount is 2125.87 US doll hairs.